terraform/modules/monitoring/alarms.tf

```diff
@@ -77,3 +77,55 @@ resource "aws_cloudwatch_metric_alarm" "sidekiq_memory" {
     ServiceName = var.sidekiq_service_name
   }
 }
+
```

Comment:
I have some thoughts on where these live prompted by thinking through review apps;

Notably, we're going to need to separate things that are per review app from things that will be shared between review apps. Since the services will be per app and the database instance will not, I think this forces us not to have these rds alarms in the same module as the ecs ones.

Which gives us two approaches - either we have two monitoring modules, one covering shared things and one covering per app things, or (I think my preferred option) we move the alarms to live alongside the things they are monitoring. Either way the sns queue / notifications setup lives in a monitoring module.

We may as well do this now rather than leaving it to be review apps work.